assets/base/i18n/en.json

```diff
@@ -1511,9 +1511,11 @@
   "mobile.loading_channels": "Loading Channels...",
   "mobile.loading_members": "Loading Members...",
   "mobile.request.invalid_response": "Received invalid response from the server.",
+  "mobile.login_options.chooseTitle": "Choose your login method",
```

Comment:
about keeping this file sorted we don't need to do anything here as the script that is going to merge this into the translation server and then submit a PR back to this repo with the latests translations will take care of that ;)